### PR TITLE
Add imglyb recipe

### PR DESCRIPTION
**Pull request by hadim**

### CODE REVIEW

**Comment by hadim** [comment]
- pyjnius: https://github.com/conda-forge/pyjnius-feedstock/pull/1
- jgo: https://github.com/conda-forge/staged-recipes/pull/7065
- scyjava: https://github.com/conda-forge/staged-recipes/pull/7066
- imagej: https://github.com/conda-forge/staged-recipes/pull/7067
- imglyb: https://github.com/conda-forge/staged-recipes/pull/7068

**Comment by hadim** [comment]
**IMPORTANT**: packages 0.3.4 has not been uploaded to pypi yet.

File: recipes/imglyb/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 99f492903a9abe03056867d3e098917bdb1baf3ddf2fa56b6fa243d93c219ccf
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vvv"
```
Comment (hanslovsky) [inline recipes/imglyb/meta.yaml]: Should we install with `--no-deps` flag?
```suggestion
  script: "{{ PYTHON }} -m pip install . -vvv  --no-deps"
```

**Comment by hadim** [comment]
@ctrueden are you ok being a maintainer?

**Comment by ctrueden** [comment]
Yes, I am OK with being a maintainer of imglyb, although for this project I defer to @hanslovsky about project decisions, since he is the project lead on it.

**Comment by hadim** [comment]
@ctrueden Here it's only about conda forge releases so it's just a matter of updating the recipe whenever a new one has been uploaded to PyPi. So no big decisions to make, only a tiny PR to make sometime.

**Comment by hadim** [comment]
@scopatz could you review and merge please?

**Pull request merged**